exercise1/do.cc

```diff
@@ -18,7 +19,8 @@ void printHelp(){
 	 << "\t--brief              Output less information" << endl
 	 << "\t--greeting <string>  A greeting string" << endl
 	 << "\t--score <number>     Set the score" << endl
-	 << "\t--threshold <number>     Set the threshold" << endl;
+	 << "\t--threshold <number>     Set the threshold" << endl
+         << "\t--name <string>      Set the name" << endl;
```

Comment:
You indentation does not match the way it is done in the lines above.
Check this out as I wrote in my slack message (`git diff` or comparison of branches on GitHub).